how-to/backups/install-gitolite.md

```diff
@@ -1,88 +1,148 @@
 (install-gitolite)=
 # How to install and configure gitolite
 
-{term}`Gitolite` provides a traditional source control management server for git, with multiple users and access rights management. 
+{term}`Gitolite` provides a traditional source control management server for git, with multiple users and access rights management.
+
+It provides access so you can add remote repos in the form of `git@yourserver:some/repo/path`.
```

Comment:
question - non blocking:

To me both dance around the bush of just saying what upstreams page says at the top (https://gitolite.com/gitolite/index.html) "Gitolite allows you to setup git hosting on a central server, with fine-grained access control and many more powerful features." how about quoting that for line one?

I see you wanted to make an example what that is used for.
How about this?:
'It allows access to the served repositories in the form of `git@yourserver:some/repo/path.'